test/Microsoft.Health.Fhir.Ingest.Template.UnitTests/CollectionContentTemplateFactoryTests.cs

```diff
@@ -24,6 +26,11 @@ public void GivenEmptyTemplateCollection_WhenCreate_ThenTemplateReturned_Test(st
         {
             var template = CollectionContentTemplateFactory.Default.Create(json);
             Assert.NotNull(template);
+

```

Comment:
Should a test to cover a bad template and assert the failure.  Also, as with the comment in the CodeValueFhirTemplateFactoryTests should verify if we have a CollectionContentTemplate with bad and good templates. the good templates still work.

Reply:
I'll update.